app/src/main/java/adapter/SoundAdapter.kt

```diff
@@ -66,37 +80,19 @@ class SoundAdapter(_SoundsList: List<Sound>, private val soundClickedListener: S
     }
 
     override fun onBindViewHolder(holder: SoundViewHolder, position: Int) {
-        holder.bind(soundsList[position], this)
-        Log.d(TAG, "VH bound $holder ${holder.button}")
+        holder.bind(mSortedList[position], this)
     }
 
     override fun onViewRecycled(holder: SoundViewHolder) {
         super.onViewRecycled(holder)
         holder.unbind(this)
-        Log.d(TAG, "VH recycled $holder")
     }
 
     fun getActiveViewHolder(sound: Sound): SoundViewHolder? {
         return activeViewHolderForSound[sound]
     }
 
-    fun add(sound: Sound) {
```

Comment:
From line 83 -> 98, it is normal that every method to add/remove sounds to `soundsList` have been removed ?

Reply:
They are useless (they could be used if we decide to let the user add/remove sounds to the list, or if we add/remove the random sound button to the list dynamically (ex: in landscape mode if the height is not large enough)